docs/configuration.md

```diff
@@ -42,9 +42,10 @@ ecosystem = "Go"
 group = "dev"
 
 # Actions to take for matching packages:
-ignore = true # Ignore this package completely, including license scanning
-license.ignore = true # Ignore the license of the package, if it is not already completely ignored at the top level
-license.override = ["MIT", "0BSD"] # Override the license of the package, if it is not ignored
+ignore = true # Ignore this package completely, including both reporting vulnerabilities and license violations
+vulnerability.ignore = true # Ignore vulnerabilities for this package, while still checking the license (if not also ignored)
+license.ignore = true # Ignore the license of the package, while still checking for vulnerabilities (if not also ignored)
+license.override = ["MIT", "0BSD"] # Override the license of the package, if it is not ignored from license scanning completely
```

Comment:
note: I think overall this reads better using the term "reporting" and "checking"

imo that reduces the expectation that these options mean the underlying actions for the matched packages don't happen at all, as that's not the case (i.e we will always include the package in the API calls to determine if there are vulnerabilities, it's just if there are we ignore them)

Reply:
Huh I thought https://github.com/google/osv-scanner/pull/1206 ignored packages from being sent to the API in the first place.

Reply:
oh yeah you're right I think, though I still think my wording here is fine because really that's an optimization - our public contract in this sense I think is still (at least right now) "we won't report" not "we don't check at all"